modules/AOW_WorkFlow/AOW_WorkFlow.php

```diff
@@ -75,6 +80,8 @@ class AOW_WorkFlow extends Basic
      */
     private function getSQLOperator($key)
```

Comment:
This function complexity getting too large. Please split this function into smaller logical units.

Reply:
Could you please give a short example on how you would do this?
I think the function is fairly short and it does a single thing. But I'm open to suggestions.

Reply:
It's ok, my mistake, sorry.